tests/zoomus/components/webinar/test_add_panelists.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import unittest
+
+from zoomus import components
+import responses
+
+
+def suite():
+    """Define all the tests of the module."""
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.makeSuite(AddPanelistsV2TestCase))
+    return suite
+
+
+class AddPanelistsV2TestCase(unittest.TestCase):
+    def setUp(self):
+        self.component = components.webinar.WebinarComponentV2(
+            base_uri="http://foo.com",
+            config={
+                "api_key": "KEY",
+                "api_secret": "SECRET",
+                "version": util.API_VERSION_2,
+            },
+        )
+
+   @responses.activate
+    def test_can_add_panelists(self):
+        responses.add(
+            responses.POST, "http://foo.com/webinar/ID/panelists",
+        )
+        response = self.component.add_panelists(panelists=[{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}])
+        self.assertEqual(
+            response.request.body, '{panelists: [{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}]}'
+        )
```

Comment:
These lines are a bit too long -- if you take a look at the rest of the codebase, we try to keep lines to about 80 characters. Can I ask you to split this on to multiple lines so it meets that standard? Thanks!

Reply:
reduced to 86, more than this would make it hard to read I think.

Reply:
Great thanks! As long as you ran the `black` code formatter and it didn't complain, that works for me... because the build will fail if the formatter doesn't approve. =)